packages/components/nodes/documentloaders/Cheerio/Cheerio.ts

```diff
@@ -127,13 +128,22 @@ class Cheerio_DocumentLoaders implements INode {
             if (!limit) limit = '10'
             else if (parseInt(limit) < 0) throw new Error('Limit cannot be less than 0')
```

Comment:
Minor Readability:: avoid multiple calls to parseInt

Reply:
You're right. Updated the code so `parseInt` is only used once.